README.md

```diff
@@ -1,14 +1,33 @@
-# Your Project Name
+# Your Project Name  temperature detector 
```

Comment:
Project is too simple and too similar with your colleagues. You can ask Herve PHANZU to your in a team with him on the street light project. If you wish to work individually please change your project idea to be more complex.